frontend/src/components/PLAYGROUND/ReportSelector/ReportTypes/index.tsx

```diff
@@ -0,0 +1,31 @@
+import { FormControlLabel, Radio } from "@mui/material";
+import React from "react";
+import { IReportCategoriesAndTypes } from "types";
+type SelectedCategory = {
+  value: string;
+  types: { label: string; value: string }[];
+};
+export const ReportTypes = ({
+  values,
+  categoriesAndTypes,
+}: {
+  values: { [key: string]: string | null };
+  categoriesAndTypes: IReportCategoriesAndTypes;
+}) => {
+  // todo: Define a good type. "Any" type temporarily permitted.
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
maybe we should just disable the 'any' type rule globally at this point.

Reply:
Yes, that would be nice but we need to go back and fix any types later, hence the TODO